corehq/apps/domain/views.py

```diff
@@ -2069,6 +2086,119 @@ def friendly_repeater_names(self):
         }
 
 
+class DomainForwardingRepeatRecords(GenericTabularReport):
+    name = 'Repeat Records'
+    base_template = 'domain/repeat_record_report.html'
+    section_name = 'Project Settings'
+    slug = 'repeat_record_report'
+    dispatcher = DomainReportDispatcher
+    ajax_pagination = True
+    asynchronous = False
+    is_bootstrap3 = True
+
+    fields = [
+        'corehq.apps.reports.filters.select.RepeaterFilter',
+        'corehq.apps.reports.filters.select.RepeatRecordStateFilter',
+    ]
+
+    def _make_view_payload_button(self, record_id):
+        return '''
+        <a
+            class="btn btn-default"
+            role="button"
+            data-record-id={}
+            data-toggle="modal"
+            data-target="#view-record-payload-modal">
+            View Payload
+        </a>
```

Comment:
sadly we return rendered HTML to the report so there is no way around this unless we do a major refactor :disappointed: